docs/platforms/dart/common/feature-flags/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Set Up Feature Flags
+sidebar_order: 7000
+description: With Feature Flags, Sentry tracks feature flag evaluations in your application, keeps an audit log feature flag changes, and reports any suspicious updates that may have caused an error.
+---
+
+<PlatformContent includePath="feature-flags/prerelease-alert" />
+
+## Prerequisites
+
+- [Sentry SDK](/platforms/dart/#configure) version `9.0.0` or higher is installed.
+
+## Enable Evaluation Tracking
+
+If you use a third-party SDK to evaluate feature flags, you can enable a Sentry SDK integration to track those evaluations. Integrations are provider specific. Documentation for supported SDKs is listed below.
+
+- [Firebase Remote Config](/platforms/dart/configuration/integrations/firebase-remote-config/)
+
+### Manual Usage
+
+Per default, the Sentry SDK adds the `FeatureFlagsIntegration`, which allows you to manually track feature flag evaluations.
```

Comment:
Does the user need to be aware of this integration class? If it's automatic and we have static function `addFeatureFlag` seems to me this integration is SDKinternals not relevant to the user (other in "how to turn off feature flags")

Reply:
no they don't need to know about it

I agree let's remove the mention of the integration